README.md

```diff
@@ -662,6 +662,14 @@ Avoid ambiguity.
 
 **Versions X** and NOT **VX** or **version X**.
 
+## Terminology
+
+### Bus
+
+We agreed that we the word `Bus` should be avoided. Some replacements incldue:
+ * When referring to the topology, use decentralized or federated (as opposed to centralized e.g. RabbitMQ)
```

Comment:
Can we find a sentence for this use? I'm not sure under what circumstances I would use decentralized vs bus. 

Also, the bracketed bit makes it sound as if we are replacing the term "centralized" but I guess you mean if the terms are Broker vs Bus or Centralized vs Bus. Is that right?